src/TwinGet.Core/Packaging/OtherStrings.cs

```diff
@@ -0,0 +1,10 @@
+﻿// This file is licensed to you under MIT license.
+
+namespace TwinGet.Core.Packaging;
+
+public static class OtherStrings
```

Comment:
Marking this as something that could be renamed in the future, NotificationStrings? StateNotificationStrings? StateStrings?

Reply:
Yeah let's change the name to something better as we progress in the future. I have a feeling things are still changing so didn't bother giving it a better name.